### PR TITLE
Misc cleanups

### PR DESCRIPTION
**Pull request by vraravam**
### Description
Misc cleanups:
1) Removed unused file: `webpack.config.base.js`
2) Clean the `out` directory when running `gulp clean` - so as to avoid any cached files causing incorrect builds.
3) Some other minor refactorings.

### Checklist:

- [x] My pull request is properly named
- [x] The changes respect the code style of the project (`$ npm run lint`)
- [x] I tested/previewed my changes locally

### CODE REVIEW

File: src/config.js [hunk 1]
```diff
@@ -33,8 +32,6 @@ export const DEVELOPMENT_TODOS_FRONTEND_URL = 'https://development--franz-todos.
 
 export const CDN_URL = 'https://cdn.franzinfra.com';
```
Comment (kris7t) [inline src/config.js:33]: Why do we still have a reference to `CDN_URL` here?

I see a usage in

https://github.com/getferdi/ferdi/blob/d16807c4935e961b6a087bf20412299ef1a7b46c/src/components/auth/SetupAssistant.js#L243

and in

https://github.com/getferdi/ferdi/blob/d16807c4935e961b6a087bf20412299ef1a7b46c/src/components/auth/SetupAssistant.js#L276

but I don't understand why can't we just load the service icons there normally.

Also, why do we have a bunch of references to Franz infra earlier in this file? (see also `src/environment.js`, which is supposed to refer to some of these constants, but in many cases, it just exports them but the exports are never used in the app elsewhere).

Comment (vraravam) [reply src/config.js:33]: I didnt unravel any other threads - except for the one related to the Google Analytics. Can we do that as part of different PR or I can do that here - just not sure how much of code will be impacted.

Comment (vraravam) [reply src/config.js:33]: These are probably left over from way back when Ferdi was hard-forked and some text was changed by find-replace.

Comment (kris7t) [reply src/config.js:33]: Sure, we can fix this in a later PR then.

**Review by kris7t — APPROVED** [review]

**Pull request merged**